src/components/checkbox/checkbox.mdx

```diff
@@ -69,8 +69,6 @@ import { Checkbox, CheckboxGroup } from "carbon-react/lib/components/checkbox";
 
 ### With labelHelp
```

Comment:
**suggestion(non-blocking)**: similar to `RadioButton`, what do you think about removing this section from the public facing docs, to reduce potential confusion over current behaviour?